Igor/Task by Andre 2.py

```diff
@@ -1010,4 +1010,79 @@
 #     else:
 #         print('No')
 # else:
-#     print('The number must be between 1 and 8')
\ No newline at end of file
+#     print('The number must be between 1 and 8')
+
+# task
+
+# import random
+#
+# def start_game():
+#     print("Hello, I am a new super game, simple as two pennies.")
+#
+#     while True:
+#         print("\nI guessed a number between 1 and 100. Try to guess it!")
+#
+#         comp_choice = random.randint(1, 100)
+#
+#         while True:
+#             try:
+#                 guess = int(input('Enter your assumption: '))
+#             except ValueError:
+#                 print('Please enter only numbers')
+#                 continue
+#
+#             if guess < comp_choice:
+#                 diff = comp_choice - guess
+#                 if diff > 10:
+#                     print("No, you're wrong, it's more than 10.")
+#                 elif diff > 5:
+#                     print("No, you're wrong, it's more than 5.")
+#                 else:
+#                     print("No, you're wrong. It's a bit more.")
+#             elif guess > comp_choice:
+#                 diff = guess - comp_choice
+#                 if diff > 10:
+#                     print("No, you're wrong, there are less than 10.")
+#                 elif diff > 5:
+#                     print("No, you're wrong, there are less than 5")
+#                 else:
+#                     print("No, you're wrong. It's a bit less.")
+#             else:
+#                 print(f"Congratulations! You guessed the number {comp_choice}. Do")
+#                 break
+#
+#         play_again = input("Do you want to play again? (yes/no): ").strip().lower()
+#         if play_again != 'yes':
+#             print("Thanks for playing! Goodbye!")
+#             break
+#
+# start_game()
+
+#task
+import os
+
+def counter(fname):
+    num_words = 0
+    num_lines = 0
+    num_charc = 0
+
+    with open(fname, 'r') as f: #f сохраняет обьект файла в пеменной
```

Comment:
You should start to write comments in English , it is more usable in EU

Reply:
When you did one task in some file , for instance Guess the number , commit it to github  and just move on to the next task  and when you done (commits and etc) create PR (when you commits separately , for example Guess number in the specific file  or text analyzing  it`s must be in different files . ) So I mean , when I`ll check you PR I`ll see all task separately